### ISSUE TITLE
[Web] probability is not returned: `error code = 1`

### ISSUE THREAD

**Issue by robingenz** [microsoft/onnxruntime#15511]
### Describe the issue

**Current behavior**: I get the following error when I try to get the prediction probability output on the web: 

```
Error: Can't access output tensor data. error code = 1
```

**Expected behavior**: I want to output the prediction probability.

### To reproduce

I have created a **reproducible example**: https://github.com/robingenz/capacitor-onnx-runtime-demo

There is also a published **web version**: https://capacitor-onnx-runtime-demo-2maes71e0-robingenz.vercel.app/home

**Steps to reproduce**:
1. Click on `Iris Species`
2. Click on `Run Inference`

The problem is this line: https://github.com/robingenz/capacitor-onnx-runtime-demo/blob/main/src/app/pages/iris-classification/iris-classification.page.ts#L59

If I remove this line, then it works without problems but I only get the `output_label`.
However, I also need access to the `output_probability`.

The same code in Python works:

```py
import onnxruntime as rt
import numpy as np

session = rt.InferenceSession("iris.onnx")

y_pred = session.run(['output_label', 'output_probability'], { 'float_input': X_test.astype(np.float32) })

print(y_pred)
```

This is the code I used to create the model:

```py
from sklearn.datasets import load_iris
from sklearn.model_selection import train_test_split
from sklearn.linear_model import LogisticRegression
from skl2onnx import convert_sklearn
from skl2onnx.common.data_types import FloatTensorType

iris = load_iris()

X_train, X_test, y_train, y_test = train_test_split(iris.data, iris.target, test_size=0.2)

clf = LogisticRegression(random_state=0)
clf.fit(X_train, y_train)

initial_type = [('float_input', FloatTensorType([None, 4]))]
onnx_model = convert_sklearn(clf, initial_types=initial_type)

with open("iris.onnx", "wb") as f:
    f.write(onnx_model.SerializeToString())
```

You can download the model [here](https://github.com/robingenz/capacitor-onnx-runtime-demo/blob/main/src/assets/models/iris.onnx).

### Urgency

The prediction probability is a critical parameter that should definitely be provided.
The release of our new application will have to wait until we get access to this parameter.
Please let me know if I can help in any way or if any information is still missing.
First of all, it would help me a lot to know if I am doing something wrong or if it is indeed a bug.

### ONNX Runtime Installation

Released Package

### ONNX Runtime Version or Commit ID

1.14.0

### Execution Provider

WASM

**Comment by robingenz** [comment]
Sorry, I forgot to publish the repo. It is now public.

**Comment by wschin** [comment]
@guschmue, @fs-eire, any comments? This is probably a traditional ML op. Not sure if WASM support it or now.

**Comment by guschmue** [comment]
We currently don't support the output type sequence<map<int64,float32>> that output_probability has.
I'd need to check what it would take to support it unless @fs-eire  already knows.

**Comment by fs-eire** [comment]
> We currently don't support the output type sequence<map<int64,float32>> that output_probability has. I'd need to check what it would take to support it unless @fs-eire already knows.

yes, you are right. currently our JS API does not support non-tensor types in model input/output.

**Comment by robingenz** [comment]
Thanks for your fast response!
Do you have a short code snippet that shows the workaround?
